src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,24 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedId, setSelectedId }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+
+        {tabs.map(tab => (
+          <li className="is-active" data-cy="Tab" key={tab.id}>
```

Comment:
you render every tab as selected 
it's incorrect 
<img width="319" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/596d22b6-56ec-407a-a8b9-d598222ace73">